src/main/java/com/project/controller/ApplicationController.java

```diff
@@ -35,18 +35,18 @@ public ApplicationController(ApplicationService applicationService) {
     }
 
     @PostMapping
-    ResponseEntity createApplication(@Valid @RequestBody Application newApplication) throws UserException, ApplicationNotFoundException {
+    ResponseEntity<HttpStatus> createApplication(@Valid @RequestBody Application newApplication) throws UserException, ApplicationNotFoundException {
```

Comment:
No need to say it's a `newApplication` calling it just `application` is good enough.

Reply:
renamed newApplication to application
made corresponding changes to UserController class